app/uk/gov/hmrc/digitalservicestax/aa_data/Period.scala

```diff
@@ -35,6 +37,12 @@ case class Period(
 
 object Period {
 
+  implicit val periodOrdering: Ordering[Period] = (x: Period, y: Period) => {
+    x.end compareTo y.end
+  }
+
+  implicit val periodOrder: Order[Period] = cats.Order.fromOrdering(periodOrdering)
```

Comment:
If you're importing `Order` here you don't need a fully qualified path to it. Also I think (though I'm struggling to find examples), it's preferred to import `cats.Order`.